nebu/cli/main.py

```diff
@@ -139,21 +146,116 @@ def get(col_id, col_version, output_dir):
     shutil.rmtree(str(tmp_dir))
 
 
-@cli.command()
-@click.argument('content_dir', default='.',
-                type=click.Path(exists=True, file_okay=False))
-def validate(content_dir):
-    content_dir = Path(content_dir).resolve()
-    struct = parse_litezip(content_dir)
+def is_valid(struct):
+    """Checks validity of a litezip's contents with commandline ouput
+    containing any validation errors.
+    Returns True when the content is value and False if not.
+
+    """
 
     has_errors = False
     cwd = Path('.').resolve()
     for filepath, error_msg in validate_litezip(struct):
         has_errors = True
         filepath = filepath.relative_to(cwd)
         logger.error('{}:{}'.format(filepath, error_msg))
+    return not has_errors
 
-    if has_errors:
+
+@cli.command()
+@click.argument('content_dir', default='.',
+                type=click.Path(exists=True, file_okay=False))
+def validate(content_dir):
+    content_dir = Path(content_dir).resolve()
+    struct = parse_litezip(content_dir)
+
+    if is_valid(struct):
+        logger.info("All good! :)")
+    else:
         logger.info("We've got problems... :(")
```

Comment:
Should this also have `sys.exit(1)`?

Reply:
Yeah, I'm unsure why I didn't add that here before. I was just going to leave it as is, because it was outside the scope of my changes. Perhaps I should put it in...